### ISSUE TITLE
Consider implementing headers-more-nginx-module

### ISSUE THREAD

**Issue by RicoFactset** [heroku/heroku-buildpack-php#430]
Hello,

Would you consider to integrate the headers-more-nginx-module module (https://github.com/openresty/headers-more-nginx-module#readme) to Nginx?

Thanks
Rico

**Comment by abarke** [comment]
Or allow an module to be installed via ENV VAR?

**Comment by RicoFactset** [comment]
> Or allow an module to be installed via ENV VAR?

Yes, that would be perfect, actually, the version of an Nginx module as well.

**Comment by RicoFactset** [comment]
Hello again, Is there already a planned date available for implementing it, if you implement it?
